router/router.go

```diff
@@ -1117,9 +1117,22 @@ func (worker *workerT) postStatusOnResponseQ(respStatusCode int, respBody string
 
 		worker.rt.failedEventsChan <- *status
 
-		if respStatusCode >= 500 {
-			timeElapsed := time.Since(firstAttemptedAtTime)
-			if respStatusCode != types.RouterTimedOutStatusCode && respStatusCode != types.RouterUnMarshalErrorCode {
+		switch respStatusCode {
+		case transformer.DROP_STATUS_CODE:
+			status.JobState = jobsdb.Aborted.State
+		case types.RouterTimedOutStatusCode:
```

Comment:
case types.RouterTimedOutStatusCode, types.RouterUnMarshalErrorCode:
 			addToFailedMap = false
 			addToAbortMap = false
Thanks for simplifying it.

Reply:
done
Thanks!
TIL